Doc/library/tkinter.rst

```diff
@@ -67,16 +67,37 @@ Or, more often::
    from tkinter import *
 
 
-.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1)
-
-   The :class:`Tk` class is instantiated without arguments. This creates a toplevel
-   widget of Tk which usually is the main window of an application. Each instance
-   has its own associated Tcl interpreter.
-
-   .. FIXME: The following keyword arguments are currently recognized:
-
-
-.. function:: Tcl(screenName=None, baseName=None, className='Tk', useTk=0)
+.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=True, sync=False, use=None)
+
+   Toplevel Tk widget, which is usually the main window of an application.
+   Each instance has its own associated Tcl interpreter.
+
+   The :class:`Tk` class is typically instantiated using all default values.
+   However, the following keyword arguments are currently recognized:
+
+   *screenName*
+      Sets the :envvar:`DISPLAY` environmental variable. (X11 only)
+   *baseName*
+      Reads the profile file :file:`{baseName}.tcl` into the Tcl interpreter
```

Comment:
Actually all these names start with a dot. `.{baseName}.tcl` etc.

It may be worth to document reading profiles separately, and only left a short reference here. There is a common between baseName and className, and there are details that should be documented.

Reply:
> Actually all these names start with a dot. .{baseName}.tcl etc.

readprofile() says:

```
    def readprofile(self, baseName, className):
        """Internal function. It reads BASENAME.tcl and CLASSNAME.tcl ... """
        ...
        class_tcl = os.path.join(home, '.%s.tcl' % className)
        class_py = os.path.join(home, '.%s.py' % className)
        base_tcl = os.path.join(home, '.%s.tcl' % baseName)
        base_py = os.path.join(home, '.%s.py' % baseName)
        ...
```

Should we also update this docstring?